code/modules/telesci/telesci_computer.dm

```diff
@@ -267,9 +267,9 @@
 		telefail()
 		temp_msg = "ERROR!<BR>Elevation is less than 1 or greater than 90."
 		return
-	if(z_co == 2 || z_co < 1 || z_co > 6)
+	if(z_co in config.admin_levels)
 		telefail()
-		temp_msg = "ERROR! Sector is less than 1, <BR>greater than 6, or equal to 2."
+		temp_msg = "ERROR! Sector is invalid! Valid sectors are [config.player_levels]."
```

Comment:
Can't print a list like this, it'll just be outputted as `list()`.

Perhaps `"ERROR! Sector is invalid! Valid sectors are [english_list(config.player_levels)]."`?